integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -365,8 +328,8 @@ void testUpdateColumnPosition() {
 
     sql(
         String.format(
-            "CREATE TABLE %s (id STRING COMMENT '', name STRING COMMENT '', age STRING COMMENT '') USING PARQUET",
-            tableName));
+            "CREATE TABLE %s (id STRING COMMENT '', name STRING COMMENT '', age STRING COMMENT '') %s",
```

Comment:
why changing this?

Reply:
The iceberg catalog does not support `using parquet`

Reply:
The iceberg catalog does not support using parquet.  So I created a `using iceberg` string in SparkIcebergCatalogIT.

Reply:
I think we could remove `using xxx`, as it's not related to the test.

Reply:
https://github.com/datastrato/gravitino/pull/2503/commits/67f79fbe5293c090c96f06f27cd7b47b383782ce

there are some problems with executing updateColumnPosition in the hive table without `using xxx`

Reply:
I could run pass the test  locally if removing `using xx`

Reply:
i'll try it by github pipeline.